spec/files/osws/osm_suite.osw

```diff
@@ -6,8 +6,9 @@
     "../../../lib/measures"
   ],
 	"file_paths": [
-    "../../files/osms",
-    "../../files"
+    "../../files/osms/in",
+    "../../files/epws",
+    ""
```

Comment:
@macumber: With the repository restructuring, a few tweaks were needed. Tests were successful.